### PR TITLE
[Fix] Deprecated Noticed on Admin Setup Wizard

### PR DESCRIPTION
**Pull request by osmansufy**
### All Submissions:

* [ ] My code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
* [ ] My code satisfies feature requirements
* [ ] My code is tested
* [ ] My code passes the PHPCS tests
* [ ] My code has proper inline documentation
* [ ] I've included related pull request(s) (optional)
* [ ] I've included developer documentation (optional)
* [ ] I've added proper labels to this pull request

### Related Pull Request(s)

* Full PR Link

### Closes 

* Closes https://github.com/getdokan/dokan-pro/issues/3303

### How to test the changes in this Pull Request:

* Steps or issue link

### Changelog entry

*Title*

Detailed Description of the pull request. What was previous behaviour
and what will be changed in this PR.

### Before Changes

Describe the issue before changes with screenshots(s).

### After Changes

Describe the issue after changes with screenshot(s).

### Feature Video (optional)

Link of detailed video if this PR is for a feature.

### PR Self Review Checklist:

* Code is not following code style guidelines
* Bad naming: make sure you would understand your code if you read it a few months from now.
* KISS: Keep it simple, Sweetie (not stupid!).
* DRY: Don't Repeat Yourself.
* Code that is not readable: too many nested 'if's are a bad sign.
* Performance issues
* Complicated constructions that need refactoring or comments: code should almost always be self-explanatory.
* Grammar errors.

### FOR PR REVIEWER ONLY:

> As a reviewer, your feedback should be focused on the idea, not the person. Seek to understand, be respectful, and focus on constructive dialog.

> As a contributor, your responsibility is to learn from suggestions and iterate your pull request should it be needed based on feedback. Seek to collaborate and produce the best possible contribution to the greater whole.

* [ ] Correct — Does the change do what it’s supposed to? ie: code 100% fulfilling the requirements?
* [ ] Secure — Would a nefarious party find some way to exploit this change? ie: everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities?
* [ ] Readable — Will your future self be able to understand this change months down the road?
* [ ] Elegant — Does the change fit aesthetically within the overall style and architecture?

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved the handling of styles and scripts in the setup wizard's header for better visual consistency, including the addition of emoji styles.
  
- **Chores**
	- Minor adjustments made to the setup wizard's methods for enhanced style management and readability without changing functionality.
	- Enhanced clarity in method parameters and improved error handling for better readability.

### CODE REVIEW

File: includes/Admin/SetupWizard.php [hunk 1]
```diff
@@ -270,8 +270,14 @@ public function setup_wizard_header() {
...
-            <?php do_action( 'admin_head' ); ?>
+            <?php
+            wp_enqueue_emoji_styles();
+            do_action( 'admin_print_styles' );
+            ?>
+            <?php
+            wp_enqueue_admin_bar_header_styles();
+            do_action( 'admin_head' );
+            ?>
             <?php do_action( 'dokan_setup_wizard_styles' ); ?>
```
Comment (mrabbani) [inline includes/Admin/SetupWizard.php:281]: Pls put all *PHP* codeblock inside a single `php` tag.

**Review by mrabbani — CHANGES_REQUESTED** [review]

File: includes/Admin/SetupWizard.php (hunk 1 above)
Comment (osmansufy) [reply includes/Admin/SetupWizard.php:281]: Working on it bhai

File: includes/Vendor/SetupWizard.php [hunk 2]
```diff
@@ -319,7 +321,7 @@ public function dokan_setup_store() {
                         <input type="text" id="calc_shipping_state" name="address[state]" value="<?php echo esc_attr( $address_state ); ?>" / placeholder="<?php esc_attr_e( 'State Name', 'dokan-lite' ); ?>">
                         <span class="error-container">
                             <?php
-                            if ( ! empty( $_POST['error_address[state]'] ) ) {
+                            if ( ! empty( $_POST['error_address[state]'] ) ) { // phpcs:ignore
```
Comment (mrabbani) [inline includes/Vendor/SetupWizard.php:324]: Assign the `$_POST` to a variable  like `$request_data` to use safely instead of ignoring *PHPCS* everywhere.

**Review by mrabbani — CHANGES_REQUESTED** [review]
- Pls fix the **PHPCS** issue in `includes/Admin/SetupWizard.php`
- Be careful, You also changed `includes/Vendor/SetupWizard.php` in https://github.com/getdokan/dokan/pull/2434.  You could merge the changes in one PR if  the scopes are same.

**Review by mrabbani — CHANGES_REQUESTED** [review]
Pls remove the existing `$_POST` usage.

**Comment by jamil-mahmud** [comment]
@mrabbani vai, can we merge this PR as well in the upcoming release? It will lead to closure one dependent issue.

**Pull request merged**